core/server/src/test/java/alluxio/master/journal/ProtoBufJournalFormatterTest.java

```diff
@@ -19,4 +29,18 @@
   protected JournalFormatter getFormatter() {
     return new ProtoBufJournalFormatter();
   }
+
+  @Test
+  public void truncatedEntryDoesntFail() throws Exception {
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    ProtoBufJournalFormatter formatter = new ProtoBufJournalFormatter();
+    formatter.serialize(JournalEntry.newBuilder().setCompleteFile(
```

Comment:
Would the case "[entry][corrupted entry][entry]" occur? If master crashes during writing the second entry, I think this might be possible. Then should we delete the corrupted intermediate entries during journal deserialization?